### PR TITLE
fix(input-autofocus): add autofocus to name field in create iteration dialog box.

### PR DESCRIPTION
**Pull request by karthikjeeyar**
- [ ] What does this PR do? 

     On opening create iteration dialog box, the mandatory name field will be auto focussed.

- [ ] What issue/task does this PR references?
    
    Fixes: https://github.com/openshiftio/openshift.io/issues/1994

### CODE REVIEW

**Comment by alien-ike** [comment]
### Ike Plugins (test-keeper)

Thank you @karthikjeeyar for this contribution!



It appears that no tests have been added or updated in this PR.

Automated tests give us confidence in shipping reliable software. Please add some as part of this change.

If you are an admin or the reviewer of this PR and you are sure that no test is needed then you can use the command `/ok-without-tests` as a comment to make the status green.


Your plugin configuration is stored in the [file](https://github.com/fabric8-ui/fabric8-planner/blob/dfdee27399fc42b9d1be78bf588dfebcaf9902ff/.ike-prow/test-keeper.yaml).

**Comment by christianvogt** [comment]
@karthikjeeyar Considering the modal mounts to the DOM only when opened, would using the attribute `autofocus` not solve the problem?

File: src/app/components_ngrx/iterations-modal/iterations-modal.component.ts [hunk 1]
```diff
@@ -232,6 +233,12 @@ export class FabPlannerIterationModalComponent implements OnInit, OnDestroy, OnC
       this.modalTitle = 'Close Iteration';
     }
     this.createUpdateIterationDialog.open();
+    if (this.iterationNameField && this.iterationNameField.nativeElement) {
+      console.log('Iteration name field' + this.iterationNameField);
```
Comment (joshuawilson) [inline src/app/components_ngrx/iterations-modal/iterations-modal.component.ts]: please remove console.log before this is merged

**Comment by vikram-raj** [comment]
@rohitkrai03 ```fabric8-planner``` and ```ngx-widgets``` are moving to ```fabric8-ui``` as a monorepo. So, we can use the autofocus directive at all the places from ```fabric8-ui```.

**Comment by karthikjeeyar** [comment]
@joshuawilson  Vikram is already working on this autofocus directive, which will get merged to fabric8-ui. I will use that directive and raise a new PR for this issue.

**Pull request closed without merging**